docs/cluster_configuration.md

```diff
@@ -257,3 +257,63 @@ For updating a manifest in an existing ACM PolicyGenerator, the following steps
       policyNamespace: ztp-clustertemplate-a-v4-16
```

Comment:
Not related to this PR! Should these namespaces be changed to `ztp-sno-ran-du-v4-16` which match the PG namespace?

Reply:
Yes, good catch. I'll update all of them in a future PR. Thank you.